lib/benchee/formatters/console/run_time.ex

```diff
@@ -235,6 +235,27 @@ defmodule Benchee.Formatters.Console.RunTime do
   defp comparison_report([_scenario], _, _, _), do: []
   defp comparison_report(_, _, _, %{comparison: false}), do: []
 
+  defp comparison_report([scenario | other_scenarios], units, label_width, %{
+         reference_job: reference_job
+       })
+       when is_binary(reference_job) do
+    %{found_job: found_job, other_scenarios: other_scenarios} =
+      find_reference_job([scenario | other_scenarios], reference_job)
+
+    scenario =
+      if found_job != nil do
```

Comment:
Usually negations should be avoid as clauses in `if` statements if possible. Many people find them a little easier to grasp that way.

So this could also be written as:

```elixir
if is_nil(found_job) do
  scenario
else
  found_job
end
```